ee/codegen/src/__test__/__snapshots__/project.test.ts.snap

```diff
@@ -676,12 +676,14 @@ exports[`WorkflowProjectGenerator > function > should generate inline workflow t
 "from vellum.workflows.nodes.displayable import FinalOutputNode
 from vellum.workflows.state import BaseState
 
-from .tool_call_get_current_weather_node import GetCurrentWeatherNode
+from .tool_call_get_current_weather_node import (
+    GetCurrentWeatherNode as ToolCallGetCurrentWeatherNodeGetCurrentWeatherNode,
```

Comment:
This is wrong. We don't need to alias in this case. We are aliasing in all cases of node references which is incorrect